.github/gitversion.yml

```diff
@@ -0,0 +1 @@
+mode: ContinuousDeployment
```

Comment:
So, what’s this? And it seems to release every commit, why?

Reply:
> So, what’s this? 

This is configuration for [GitVersion](https://gitversion.net/) so the `fullSemVer` variable used for versioning is incremented on every commit, thus avoiding conflicting version numbers.

> And it seems to release every commit, why?

Well, it increments the version number according to the number of commits since the last `git tag`. But there's no actual release being performed. Seeing how we only do `publish` on `release` now, we don't really need this anymore, but there's no harm keeping it either.